src/components/copyText/test/copyText.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { act, fireEvent, render, screen } from '@testing-library/react';
+import CopyText, { copyToClipboard } from '../index';
+import React from 'react';
+
+Object.assign(navigator, {
+  clipboard: {
+    writeText: jest.fn(),
+  },
+});
+
+describe('copyText tests', () => {
+  describe('copyToClipboard test', () => {
+    it('writes to clipboard', async () => {
```

Comment:
In the future, you dont need to test something like this, but im glad you did anyway!